CHANGELOG.md

```diff
@@ -8,37 +8,92 @@ Changelog
   * acl (@saqibraza)
   * ace (@yjyongz)
   * remark ace (@bansalpradeep)
+  * **Platforms:** Nexus (3|9)k
 * EVPN
   * evpn_vni (@andish)
+  * **Platforms:** Nexus (3|9)k
```

Comment:
For compactness, did you consider appending the platforms to the feature name line instead of having a separate line?

```markdown
* EVPN (platforms: Nexus 3k and Nexus 9k)
  * evpn_vni (@andish)
```